cosmo_tester/framework/cfy_helper.py

```diff
@@ -97,6 +101,34 @@ def bootstrap(self,
                 task_retry_interval=task_retry_interval,
                 verbose=verbose,
                 debug=debug).wait()
+            self.upload_plugins(cfy)
```

Comment:
Just checked the code, and it appears I was mistaken. Actual process of simple manager blueprint test:
1. Create a VM on openstack.
2. Run cfy bootstrap locally using the simple manager blueprint and setting the VM IP as public_ip, which causes the simple manager blueprint to bootstrap on that machine via fabric. It does this using cfy_helper's bootstrap method.